frodo.js

```diff
@@ -9,40 +9,44 @@
  * Copyright 2016 Henrik Ingo (@henrikingo)
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!